dags/aps/fetch_from_api.py

```diff
@@ -0,0 +1,50 @@
+import io
+from datetime import date, datetime
+from airflow.models import Variable
+import os
+import json
+
+from aps.APSParams import APSParams
+from airflow.api.common import trigger_dag
+
+
+def set_APS_harvesting_interval(repo):
+    repo.delete_all()
+    today = date.today()
+    aps_fetching_from_date = repo.find_the_last_uploaded_file_date()
+    aps_fetching_until_date = today.strftime("%Y-%m-%d")
+    if aps_fetching_from_date is None:
+        Variable.set('aps_fetching_from_date', aps_fetching_until_date)
+    else:
+        Variable.set('aps_fetching_from_date', aps_fetching_from_date)
+    Variable.set('aps_fetching_until_date', aps_fetching_until_date)
+    return({'aps_fetching_from_date': Variable.get('aps_fetching_from_date'), 'aps_fetching_until_date': Variable.get('aps_fetching_until_date')})
+
+
+def save_file_in_s3(rest_api, repo):
+    articles_metadata = rest_api.get_articles_metadata()
+    if bool(articles_metadata):
+        date_today =  date.today().strftime("%Y-%m-%d")
+        prefix = date.today().strftime("%Y-%m-%d_%H:%M:%S")
+
+        byte_file = io.BytesIO(articles_metadata)
+        key = '/'.join([date_today, '.'.join([prefix, 'json'])])
+        repo.save(key, byte_file)
+        return key
+        
+def split_json(repo, key):
+    file = repo.find_by_id(key)
+    data = json.loads(file.getvalue().decode('utf-8'))['data']
+    ids_and_articles=[]
+    for article in data:
+        doi = article['identifiers']['doi']
+        today =  datetime.now().strftime('%Y-%m-%dT%H:%M:%S.%f')
+        id = '_'.join([f'APS_{doi}_', today])
```

Comment:
This is wrong, it will return 
```APS_DOI__2022-01-01:XXXXXXX```

```suggestion
        id = f"APS_{doi}_{today}"
```